docs/DesignDocs/CustomAttestation.md

```diff
@@ -764,7 +804,7 @@ oe_result_t oe_unregister_verifier(
  * @retval OE_NOT_FOUND The plugin does not exist.
  */
 oe_result_t oe_get_evidence(
-    const uuid_t* evidence_format_uuid,
+    const uuid_t* evidence_format_id,
```

Comment:
What is the relationship between the evidence_format_id and the format_id of the plugins? (i.e. why call them different things?)